mixer/pkg/il/compiler/compiler_test.go

```diff
@@ -1189,91 +67,27 @@ func TestCompile(t *testing.T) {
 				}
 			}
 
-			// TODO: replace with GetMutableBagForTesting()
-			b := iltest.FakeBag{Attrs: te.input}
-
-			ipExtern := interpreter.ExternFromFn("ip", func(in string) []byte {
-				if ip := net.ParseIP(in); ip != nil {
-					return []byte(ip)
-				}
-				return []byte{}
-			})
-
-			ipEqualExtern := interpreter.ExternFromFn("ip_equal", func(a []byte, b []byte) bool {
-				// net.IP is an alias for []byte, so these are safe to convert
-				ip1 := net.IP(a)
-				ip2 := net.IP(b)
-				return ip1.Equal(ip2)
-			})
-
-			var timestampExternFn = interpreter.ExternFromFn("timestamp", func(in string) time.Time {
-				layout := time.RFC3339
-				t, _ := time.Parse(layout, in)
-				return t
-			})
-
-			var timestampEqualExternFn = interpreter.ExternFromFn("timestamp_equal", func(t1 time.Time, t2 time.Time) bool {
-				return t1.Equal(t2)
-			})
-
-			externMap := map[string]interpreter.Extern{
-				"ip":              ipExtern,
-				"ip_equal":        ipEqualExtern,
-				"timestamp":       timestampExternFn,
-				"timestamp_equal": timestampEqualExternFn,
+			input := test.I
+			if input == nil {
+				input = map[string]interface{}{}
 			}
+			b := ilt.FakeBag{Attrs: input}
 
-			i := interpreter.New(result.Program, externMap)
+			i := interpreter.New(result.Program, runtime.Externs)
 			v, err := i.Eval("eval", &b)
 			if err != nil {
-				if len(te.err) != 0 {
-					if te.err != err.Error() {
-						tt.Fatalf("expected error not found: E:'%v', A:'%v'", te.err, err)
-					}
-				} else {
-					tt.Fatal(err)
+				if test.Err != err.Error() {
+					tt.Fatalf("expected error not found: E:'%v', A:'%v'", test.Err, err)
 				}
 				return
 			}
-
-			if len(te.err) != 0 {
-				tt.Fatalf("expected error not received: '%v'", te.err)
+			if test.Err != "" {
+				tt.Fatalf("expected error not received: '%v'", test.Err)
 			}
 
-			// Byte arrays are not comparable natively
-			bExp, found := te.result.([]byte)
-			if found {
-				bAct, found := v.AsInterface().([]byte)
-				if !found || !bytes.Equal(bExp, bAct) {
-					tt.Fatalf("Result match failed: %+v == %+v", v.AsInterface(), te.result)
-				}
-			} else if v.AsInterface() != te.result {
-				tt.Fatalf("Result match failed: %+v == %+v", v.AsInterface(), te.result)
+			if !ilt.AreEqual(test.R, v.AsInterface()) {
+				tt.Fatalf("Result match failed: %+v == %+v", test.R, v.AsInterface())
 			}
 		})
 	}
 }
-
-func TestCompile_ParseError(t *testing.T) {
```

Comment:
Are these tests mapped in the new test data ? And the below one too.

Reply:
Yes, they are both captured in the test set:
https://github.com/ozevren/istio/blob/08e84650e43f30bdbb9001561b7076e0252a7b0e/mixer/pkg/il/testing/tests.go#L1444

https://github.com/ozevren/istio/blob/08e84650e43f30bdbb9001561b7076e0252a7b0e/mixer/pkg/il/testing/tests.go#L79